SamKTech/Sanum
# Synced Version of Codebase

## **Types of Changes**

## **Purpose**
- Implement log out button feature, input and showing caloric goal of user
- Give some styling to our web application
- Fix merge conflicts and bugs in codebase

## **Approach**
- Created Redirect routing and changing state to successfully log in
- Integrated post request to backend to store the caloric goal of the user and render it to the frontend
- Utilized scss for styling

## **How to Test**
- npm install
- npm run dev
- on the Profile page, check if inputting the caloric goal works (look at console)

## **Screenshot(s)**
**Profile Page with Caloric Goal**
![Screen Shot 2021-04-01 at 12 30 16 AM](https://user-images.githubusercontent.com/75049208/113259067-91d4a200-9281-11eb-847c-4da57022b126.png)

**Feed Page with styling**
![Screen Shot 2021-04-01 at 12 30 57 AM](https://user-images.githubusercontent.com/75049208/113259073-926d3880-9281-11eb-94aa-9a4d358a394c.png)

Collaborated and Co-Authored:
@khuongdn16
@koderka2020
@tedtaemin
